app/javascript/components/home/LeftCol.jsx

```diff
@@ -12,9 +12,9 @@ class LeftSideHome extends React.Component {
             adipiscing elit, sed do eiusmod tempor incididunt ut labore et
             dolore magna aliqua.
           </p>
-          <button className="btn wide-btn btn-dark" type="button">
+          <Link to={"/specifications"} className="btn wide-btn btn-dark">
```

Comment:
Strings don't have to be evaluated in JSX, so curly braces aren't necessary here.